project/Dependencies.scala

```diff
@@ -67,6 +67,16 @@ object Dependencies {
       )
   )
 
+  val DiscoveryConsul = Seq(
+    libraryDependencies ++=
+      DependencyGroups.AkkaActor ++
+      DependencyGroups.AkkaTesting ++
+      Seq(
+        "com.orbitz.consul" % "consul-client" % "1.1.2",
+        "com.pszymczyk.consul" % "embedded-consul" % "1.0.2" % "test"
+      )
```

Comment:
Please add a comment with the license next to each of those lines, we have to make sure they are apache compatible (i.e. no GPL code)

Reply:
Added